src/main/java/by/exadel/internship/team_four/entity/enums/FormatOfInternship.java

```diff
@@ -0,0 +1,6 @@
+package by.exadel.internship.team_four.entity.enums;
+
+public enum FormatOfInternship {
```

Comment:
Please, replace everywhere you use genitive nouns

```suggestion
public enum InternshipFormat {
```

Reply:
It's not resolved

Reply:
sorry, it's done now